lib/sisimai/bite/email/exchange2007.rb

```diff
@@ -0,0 +1,178 @@
+module Sisimai::Bite::Email
+  module Exchange2007
+    # Sisimai::Bite::Email::Exchange2007 parses a bounce email which created by
+    # Microsoft Exchange Server 2007.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Exchange2007.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'subject'          => %r/\AUndeliverable:/,
```

Comment:
__[rubocop]__
* Style/SymbolLiteral: Do not use strings for word-like symbol literals.
* Layout/SpaceAroundOperators: Operator `=>` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->